.mypy.ini

```diff
@@ -0,0 +1,67 @@
+[mypy]
+files = lib/stitches/**/*.py
+python_version = 3.8
+
+# error: "None" not callable  [misc] shows up a lot, disabling to start
+disable_error_code = misc
```

Comment:
This can be removed once all those errors are fixed (or not). I think it could be addressed by checking for None before attempting to call them, and annotating the types as Optional[foo]